miqi/skills/qraft-workflowspec-export/scripts/auth.py

```
@@ -266,13 +266,9 @@ def _try_json(resp: httpx.Response) -> dict[str, Any]:
 def exchange_token(
     client: httpx.Client, base_url: str, code: str, redirect_uri: str
 ) -> dict[str, Any]:
-    # client_secret 不落仓库/技能包：要求显式配置，缺失即分类报错（#674 脱敏要求）。
-    client_secret = os.environ.get("QRAFT_CLIENT_SECRET", "").strip()
-    if not client_secret:
-        raise AuthError(
-            "CLIENT_SECRET_MISSING",
-            "缺少 QRAFT_CLIENT_SECRET 环境变量，无法换取 token（凭据不硬编码在脚本内）",
-        )
+    # 测试阶段开箱即用：硬编码默认值，QRAFT_CLIENT_SECRET 环境变量可覆盖
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)